packages/lib/services/plugins/defaultPlugins/defaultPluginsUtils.ts

```diff
@@ -15,16 +18,25 @@ export function checkPreInstalledDefaultPlugins(defaultPluginsId: string[],plugi
 	}
 }
 
-export async function installDefaultPlugins(service: PluginService, pluginsDir: string, defaultPluginsId: string[], pluginSettings: PluginSettings): Promise<PluginSettings> {
-	const defaultPluginsPaths = await shim.fsDriver().readDirStats(pluginsDir);
+export async function installDefaultPlugins(service: PluginService, defaultPluginsDir: string, defaultPluginsId: string[], pluginSettings: PluginSettings): Promise<PluginSettings> {
+	if (!await shim.fsDriver().exists(defaultPluginsDir)) {
+		logger.info('Could not find default plugins\' directory - skipping installation.');
+		return pluginSettings;
+	}
+	const defaultPluginsPaths = await shim.fsDriver().readDirStats(defaultPluginsDir);
+	if (defaultPluginsPaths.length <= 0) {
+		logger.info('Default plugins\' directory is empty - skipping installation.');
```

Comment:
add the path of the directory to the message

Reply:
Done :+1: